src/pybind/mgr/dashboard/services/cluster.py

```diff
@@ -44,6 +48,35 @@ def from_db(cls):
     @classmethod
     def get_capacity(cls) -> ClusterCapacity:
         df = mgr.get('df')
+        total_objects = 0
+        total_pool_bytes_used = 0
+        average_object_size = 0
+        rgw_pools = cls.get_rgw_pools()
+        for pool in df['pools']:
+            if pool['name'] in rgw_pools:
+                total_objects = total_objects + pool['stats']['objects']
+                total_pool_bytes_used = total_pool_bytes_used + pool['stats']['bytes_used']
+        if total_objects and total_objects != 0:
+            average_object_size = total_pool_bytes_used / total_objects
         return ClusterCapacity(total_avail_bytes=df['stats']['total_avail_bytes'],
                                total_bytes=df['stats']['total_bytes'],
-                               total_used_raw_bytes=df['stats']['total_used_raw_bytes'])._asdict()
+                               total_used_raw_bytes=df['stats']['total_used_raw_bytes'],
+                               total_objects=total_objects,
+                               total_pool_bytes_used=total_pool_bytes_used,
+                               average_object_size=average_object_size)._asdict()
+    @classmethod
+    def get_rgw_pools(cls):
+        pool_names = []
+        rgw_pool_names = []
+        ret, out, _ = mgr.check_mon_command({
```

Comment:
We already have a pool list API in dashboard. Why not use that and filter based on `application_metadata == rgw`

Reply:
Updated the PR